### PR TITLE
rename test file so it gets picked up

### PR DESCRIPTION
**Pull request by mrmasterplan**
this test file had a spelling error in its name and was not picked up by the test  pipeline. It is tested now.

### CODE REVIEW

**Comment by LauJohansson** [comment]
I approve this - when you have fixed the test - you can merge.

**Review by LauJohansson — DISMISSED** [review]

File: tests/cluster/delta/test_sparkexecutor.py
Comment (LauJohansson) [inline tests/cluster/delta/test_sparkexecutor.py:10]: If we parralize test jobs - this will end up intefere with other tests. 
Also - in a true dataplatform setup - this CAN NOT be used since it will drop true databases

**Comment by LauJohansson** [comment]
> I approve this - when you have fixed the test - you can merge.

I have removed my previous approval. More changes has happened to core logic.

**Comment by mrmasterplan** [comment]
In another feature I am developing I wanted the ability to inline the contents of an sql file. I took the `get_statements` function, which was quite long anyway and split it into two parts, `_get_raw_contents` finds and reads all the sql files. then `chop_and_substitute` takes there raw contents, divides into statements and executes the substitutions. The functionality of all existing methods has not been affected.

**Pull request merged**
